tech: add snapshot test for add ons component

anusha5695 opened the pull request:
## Related to issue: https://github.com/rahuldkjain/github-profile-readme-generator/issues/294

Snapshot test for addons component

anusha5695 commented:
@rahuldkjain  added snapshot test :)

anusha5695 commented:
@rahuldkjain could you review this please?

Citrinin commented on src/components/__tests__/addons.test.js:
```diff
@@ -0,0 +1,95 @@
+import React from "react"
+import renderer from "react-test-renderer"
```
Thanks for your efforts, looks good!
But we shall rely only on one testing library use `enzyme-to-json` library and do not use `react-test-renderer`  - there is no trace of it in `master` now. Check how to use it in other test files.

anusha5695 replied:
Hi @Citrinin done

anusha5695 commented:
Hi @rahuldkjain , fixed merge conflicts and added 20 more tests to increase coverage as just snapshot test for the comment wasn't sufficient. Could you check plz?

zenby reviewed (approved):
Great @anusha5695 you've done a great job! 
LGTM! :+1:

anusha5695 commented:
Thank you @zenby ! 👍
